frame/staking-rewards/src/lib.rs

```diff
@@ -417,32 +432,16 @@ pub mod pallet {
 				},
 			};
 			RewardPools::<T>::insert(
-				T::RewardPoolId::from(T::PicaAssetId::get().into()),
+				T::RewardPoolId::from(staked_asset_id.into()),
 				pica_staking_pool,
 			);
-			// PBLO staking pool
-			let pblo_staking_pool: RewardPoolOf<T> = RewardPool {
-				owner: T::PalletId::get().into_account_truncating(),
-				asset_id: T::PbloAssetId::get(),
-				rewards: Default::default(),
-				total_shares: T::Balance::zero(),
-				claimed_shares: T::Balance::zero(),
-				end_block: T::BlockNumber::zero(),
-				lock: LockConfig {
-					duration_presets: [
-						(ONE_WEEK, Perbill::from_percent(1)),
-						(ONE_MONTH, Perbill::from_percent(10)),
-					]
-					.into_iter()
-					.try_collect()
-					.expect("Genesis config must be correct; qed"),
-					unlock_penalty: Default::default(),
-				},
-			};
-			RewardPools::<T>::insert(
-				T::RewardPoolId::from(T::PbloAssetId::get().into()),
-				pblo_staking_pool,
-			);
+			T::FinancialNft::create_collection(
```

Comment:
pallet_staking_rewards is initialized before pallet_fnft. This may create undefined behavior

Reply:
For now I will move the fNFT pallet to be defined before staking rewards